### DISCUSSION TITLE
Como se manter motivado nos estudos?!

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#439] [Discussão]
Opa galera beleza? Estou enfrentando um problema de motivação nos estudos, me sentindo meio perdido, sei que é um problema pessoal mas acredito que alguém já tenha passado por isso, oq vcs fazem/usam para se manterem motivados? Alguma metodologia legal que ajude nos estudos?

**Comment by wellingtonpgp** [comment]
@Cleysonlb cara eu costumo sempre que vou estudar algo novo ter um projeto em mente para utilizar o que vou estudar projeto simples nada muito complexo para não desmotivar, e uma metodologia que uso e estudar uma semana direto aquela tecnologia focado apenas nela, vendo diversas coisa sobre aquele assunto, como artigos, dicas, videos etc. O maior problema em estudos costuma ser foco, para manter o foco utilizo a técnica do pomodoro e também esse site https://habitica.com tipo você lança suas tarefas e quando concluídas vai evoluindo seu avatar no site, e vai subindo de nível e mais uma motivação para continuar estudando. Comigo está funcionado bem. :nerd_face:

**Comment by gabrieldarezzo** [comment]
Rolou um podcast bem bacana, é uma das pautas era exatamente sua issue.

http://devnaestrada.com.br/2016/03/18/devnaestrada-entrevista-rodrigo-franco.html

**Comment by fernandosouza** [comment]
Procuro trabalhar da mesma forma que o @wellingtongeek. Infelizmente estou numa fase onde tenho de estudar muita coisa em um periodo de tempo relativamente curto.

Mas realmente a melhor forma de aprender a codificar é codificando. Você pode fazer um projetinho bacana e a cada nova tecnologia você ou pode fazer melhorias nesse repositório ou pode reescrever as features em um novo framework, se assim for o caso.

**Comment by thulioph** [comment]
Venho seguindo as práticas citadas pelo pessoal, a técnica do Pomodoro é massa, você consegue fazer tudo que planeja e administra melhor o estudo.

To tentando adicionar as minhas práticas a escrita, além de fazer um projeto pequeno, eu vou tentando escrever um post sobre como eu fiz, porque utilizei, como funciona, etc.. 

Escrever ajuda a fixar o conteúdo e quem sabe ajuda alguém que também tá começando.

**Comment by torto** [comment]
Ser jovem! Vc vai ficando velho vc vai perdendo a disposição.. por isso vc não vê muitos programadores com 40 anos no mercado!

**Comment by matheus-vieira** [comment]
> por isso vc não vê muitos programadores com 40 anos no mercado!

@torto Creio que isso seja algo de brasileiro....

**Comment by bernardodiasc** [comment]
hmm, ao invés de ser generalista e afirmar algo assim, vamos olhar a coisa mais meticulosamente, veja https://www.youtube.com/watch?v=ecIWPzGEbFc ("Uncle" Bob Martin - "The Future of Programming")

a idade média nao é coisa de brasileiro e tambem nao é uma regra.
